bigtable/client/client_options_test.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/client_options.h"
+
+#include <gmock/gmock.h>
+#include <google/bigtable/v2/bigtable_mock.grpc.pb.h>
+
+TEST(ClientOptionsTest, ClientOptionsDefaultSettings){
```

Comment:
Please add a space after `)` and before `{`, similarly below.

Reply:
Done.